{"name": "test_check_instagram", "status": "failed", "statusDetails": {"message": "AssertionError: assert 301 == 200\n +  where 301 = <bound method Home.check_social_media of <Pages.homePage.Home object at 0x7feae63144f0>>('/html/body/div[2]/div[1]/div[2]/div[1]/div/div/div[3]/div/div[1]/a[4]')\n +    where <bound method Home.check_social_media of <Pages.homePage.Home object at 0x7feae63144f0>> = <Pages.homePage.Home object at 0x7feae63144f0>.check_social_media\n +      where <Pages.homePage.Home object at 0x7feae63144f0> = Home(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"822ff6353031b6da1bcde5c3b3b7ae15\")>)\n +        where <selenium.webdriver.chrome.webdriver.WebDriver (session=\"822ff6353031b6da1bcde5c3b3b7ae15\")> = <tests.test_social_media.TestSocialMedia object at 0x7feae64282e0>.driver\n +    and   '/html/body/div[2]/div[1]/div[2]/div[1]/div/div/div[3]/div/div[1]/a[4]' = Locators.instagram", "trace": "self = <tests.test_social_media.TestSocialMedia object at 0x7feae64282e0>\n\n    @pytest.mark.social_media\n    def test_check_instagram(self):\n>       assert Home(self.driver).check_social_media(Locators.instagram) == 200\nE       assert 301 == 200\nE        +  where 301 = <bound method Home.check_social_media of <Pages.homePage.Home object at 0x7feae63144f0>>('/html/body/div[2]/div[1]/div[2]/div[1]/div/div/div[3]/div/div[1]/a[4]')\nE        +    where <bound method Home.check_social_media of <Pages.homePage.Home object at 0x7feae63144f0>> = <Pages.homePage.Home object at 0x7feae63144f0>.check_social_media\nE        +      where <Pages.homePage.Home object at 0x7feae63144f0> = Home(<selenium.webdriver.chrome.webdriver.WebDriver (session=\"822ff6353031b6da1bcde5c3b3b7ae15\")>)\nE        +        where <selenium.webdriver.chrome.webdriver.WebDriver (session=\"822ff6353031b6da1bcde5c3b3b7ae15\")> = <tests.test_social_media.TestSocialMedia object at 0x7feae64282e0>.driver\nE        +    and   '/html/body/div[2]/div[1]/div[2]/div[1]/div/div/div[3]/div/div[1]/a[4]' = Locators.instagram\n\ntests/test_social_media.py:15: AssertionError"}, "start": 1662786450098, "stop": 1662786450824, "uuid": "f86bc318-d016-45b8-be09-f26433c9d5de", "historyId": "6aecfb3fd6d36bd94578abb01ae0d358", "testCaseId": "0197e4ff4bd19694f8bf82b88faa0b24", "fullName": "tests.test_social_media.TestSocialMedia#test_check_instagram", "labels": [{"name": "tag", "value": "social_media"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_social_media"}, {"name": "subSuite", "value": "TestSocialMedia"}, {"name": "host", "value": "mm-MS-7B89"}, {"name": "thread", "value": "18322-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_social_media"}]}